Sources/ServiceKit/Environment/Proccess.swift

```diff
@@ -0,0 +1,60 @@
+import Foundation
+
+extension Environment {
+    
+    /// The process information of an environment. Wraps `ProcessInto.processInfo`.
+    @dynamicMemberLookup public struct Process {
+        
+        /// The process information of the environment.
+        public let info: ProcessInfo
```

Comment:
We could keep this private (and maybe underscore it?). That would be useful incase anyone does `export info=foo`. 

```suggestion
        private let _info: ProcessInfo
```